ampd/proto/ampd.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+import "google/protobuf/any.proto";
+
+package ampd;
+
+message SubscribeRequest {
+  string event_type = 1;
+  map<string, string> event_attributes = 2;
```

Comment:
```suggestion
  repeated Event events_filter = 1;
```

Reply:
Interesting thought, but we are not going to do a strict comparison though. The filter may only provide a subset of the matching event.